### PR TITLE
Upgrade to aXe 2.6.1 and add Cypress tests

### PR DESCRIPTION
**Pull request by marcysutton**
I added Cypress to react-axe to ensure the integration works. @dylanb, is there any other test coverage you'd like to see? I wasn't seeing any re-renders with the existing functionality in the example app, so I didn't want to go too much farther down a refactoring rabbit-hole. This is prep work for the move to axe-core@3.0.

The two tests I added can be found in `cypress/integration`. There are two new npm scripts you can run:
```
npm test
npm run test:debug
```
The former will run headless, while the latter will open the Cypress app for debugging. I added them both to the README.

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "devDependencies": {
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
+    "browserify": "^15.2.0",
```
Comment (dylanb) [inline example/package.json]: Why did you add this dependency here?

Comment (marcysutton) [reply example/package.json]: The tests were failing for me without it after a fresh install. But I see now it should be installed globally...my global modules got messed up after a node reinstall. I'll remove it :)

File: cypress/integration/index-test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+const axe = require('../../index')
+
+describe('React-axe', function () {
+	it('should assert that page content is correct', function () {
```
Comment (WilcoFiers) [inline cypress/integration/index-test.js]: Mixed spaces / tabs

File: example/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,7460 @@
+{
```
Comment (WilcoFiers) [inline example/package-lock.json]: This file should probably be excluded.

**Comment by marcysutton** [comment]
I'm having a heck of a time trying to get the example to build without `browserify` included in package.json. I installed it globally with `npm install browserify -g` to replace the missing module on my machine after a node reinstall, but now every time I try to build the example I get `>> Error: Cannot find module 'browserify'`. Can someone else try installing/building as well as running the tests to see if it's just a problem with my machine?

**Comment by marcysutton** [comment]
Ok, after pairing with Wilco I saw that he has the same `Error: cannot find module browserify` problem on his machine with a global install. Rather than spend a bunch of time debugging dependencies in the example app (which seems to be a bug with npm and grunt-browserify), we decided to just include browserify as a dependency like I had it in the first place. This PR has been updated to reflect that decision. I need this merged to move on to aXe 3 work.
